### DISCUSSION TITLE
Option to make store persistent

### DISCUSSION THREAD

**Discussion by nathantaal** [TanStack/store#150] [Ideas]
It would be great if store got an option to make state persistant. Initially the storage mechanism does not have to be configurable. Persistent stores take away the hassle of having to save everything thing important in the store to localstorage and then retrieving and setting it at application hoot again.

**Comment by crutchcorn** [comment]
Agreed. Something we'll place on our roadmap 100%. Feel free to make a GH issue for this in the meantime

**Reply by nathantaal** [reply]
Nice, though, the issue tracker linked me here and I can open the issue as a bug, but that forces me fill most questions with N/A. Is that okay to?

**Reply by crutchcorn** [reply]
Yup, that's fine

**Reply by XDean** [reply]
I believe load persistent value requires a validation framework, right?
If yes, do you have a prefer one, so I can use the same one to reduce future migration effort. (Now, I'm using Zod)
